### PR TITLE
PXB-3322 In the 8.0 series, server version and backup version check up…

### PR DESCRIPTION
**Pull request by patrickbirch**
…date

	modified:   docs/server-backup-version-comparison.md
	modified:   docs/supported-versions.md

### CODE REVIEW

File: docs/server-backup-version-comparison.md [hunk 1]
```diff
@@ -1,50 +1,38 @@
...
+## Version updates
 
-## Version check
+Percona XtraBackup 8.0.34: Disabled the server-version-check.
 
-Before the backup starts, XtraBackup compares the source system version to
-the Percona XtraBackup version. If the source system version is greater than the XtraBackup version, XtraBackup stops the backup and returns an
-error message. This comparison prevents a failed or corrupted
-backup due to source system changes.
+Percona XtraBackup 8.0.22-15: introduced `--no-server-version-check`. This option allows you to override the version comparison process. This option is off by default, meaning the comparison occurs unless explicitly overridden.
```
Comment (alina-derkach-oaza) [inline docs/server-backup-version-comparison.md:21]: Here:
Percona XtraBackup 8.0.34: `Disabled `

and here

Percona XtraBackup 8.0.22-15: `introduced `

Make them both start from lowercase

File: docs/supported-versions.md [hunk 2]
```diff
@@ -1,9 +1,22 @@
...
+
+Specifically, Percona XtraBackup {{release}} supports taking backups of the following MySQL-compatible databases:
 
 * Percona Server for MySQL 8.0.34, and MySQL 8.0.34
 
 * Percona Server for MySQL 8.0.35, and MySQL 8.0.35
 
 * Percona Server for MySQL 8.0.36 and MySQL 8.0.36 (and any later versions in the 8.0 series)
+
+
```
Comment (satya-bodapati) [inline docs/supported-versions.md]: From here, we can provide the link to the version comparison/reference page and ask the user to look.

**Review by satya-bodapati — CHANGES_REQUESTED** [review]
rest all looks good

File: docs/supported-versions.md (hunk 2 above)
Comment (patrickbirch) [reply docs/supported-versions.md]: We do - it's in the additional resources.

Comment (satya-bodapati) [reply docs/supported-versions.md]: I mean after the  line, "Percona Server for MySQL 8.0.36 and MySQL 8.0.36"

Please add some text like, "For additional information on the version compatibility, Please check here [a link to  Server version and backup version comparison[¶](https://percona-xtrabackup-docs-8-0-pr-302.onrender.com/server-backup-version-comparison.html#server-version-and-backup-version-comparison)

Comment (patrickbirch) [reply docs/supported-versions.md]: Done

**Review by alina-derkach-oaza — APPROVED** [review]

File: docs/supported-versions.md (hunk 2 above)
Comment (satya-bodapati) [reply docs/supported-versions.md]: I dont see it @patrickbirch

**Review by satya-bodapati — APPROVED** [review]
LGTM!

**Pull request merged**